config/config_template.yaml

```diff
@@ -63,8 +68,8 @@ training:
   num_trn_samples:
   num_val_samples:
   num_tst_samples:
-  batch_size: 2
-  num_epochs: 1
+  batch_size: 4
```

Comment:
maybe keep the `batch_size` at 2 and the `num_epochs` at 1 just for people who want to test out of the box for the tamplete